crates/rolldown/src/utils/chunk/render_chunk_imports.rs

```diff
@@ -101,11 +101,15 @@ pub fn render_chunk_imports(
 
   imports_from_external_modules.iter().for_each(|(importee_id, named_imports)| {
     let importee = &graph.module_table.external_modules[*importee_id];
+    let external_module_side_effects = &importee.side_effects;
     let mut is_importee_imported = false;
     let mut import_items = named_imports
       .iter()
       .filter_map(|item| {
         let canonical_ref = graph.symbols.par_canonical_ref_for(item.imported_as);
+        if !graph.used_symbol_refs.contains(&canonical_ref) {
```

Comment:
`use_symbols_refs` should be removed. Will this block https://github.com/rolldown/rolldown/pull/1449 ?

Reply:
I try use `stmt_info.is_included` and it is not work.

Reply:
Good question. I added to my note.

Reply:
Let's merge it first, I am ok to remove all the `used_symbol_refs` if we find a better way to do this.